### ISSUE TITLE
TypeScript intellisense auto import not working with left click on suggested items

### ISSUE THREAD

**Issue by hoshinokanade** [microsoft/vscode#75949]
- VSCode Version: 1.35.1 (June build)
- OS Version: Windows 10 x64 10.0.17134

Steps to Reproduce:

1. Get a typescript file
2. Try to enter a function to make TypeScript intellisense to suggest.\
In this example I use `readfil` so that I can get `readFile` from `fs`, but it could be any name that require an auto import, e.g. from node_modules or from your code. 

![image](https://user-images.githubusercontent.com/32830686/59971372-e24fa380-95b5-11e9-88f7-2b048cc57ff9.png)
(ps: Chinese translation: `auto import from 'fs'`)

3. Mouse left click on any of the suggestion item. It will do the auto-completion but no auto-import. If you press `tab` or `enter` instead, you will get auto-completion, plus an `import { readFile } from "fs";` statement inserted at line 1.

Expected behavior: Mouse left click on any TypeScript intellisense suggestion item results the same as `tab` or `enter`. This means not only auto-completion, but also auto-import.\
Resulted behavior: Mouse left click on any TypeScript intellisense suggestion item results in auto-completion only.

Does this issue occur when all extensions are disabled?: **Yes**

**Comment by hoshinokanade** [comment]
Tried with the insider build, the issue disappears
```
Version: 1.36.0-insider (user setup)
Commit: 9ff8ae037e8e6109d65e4b5e3eb3dc60cc187e21
Date: 2019-06-21T07:52:00.716Z
Electron: 4.2.4
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.17134
```

**Comment by mjbvz** [comment]
Closing this as it sounds like it is fixed in the current insiders builds (which will be released as VS Code 1.36)
